openmetadata-spec/src/main/resources/json/schema/entity/data/query.json

```diff
@@ -0,0 +1,108 @@
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "fullyQualifiedName": {
+      "description": "Fully qualified name of a query.",
+      "$ref": "../../type/basic.json#/definitions/fullyQualifiedEntityName"
+    },
+    "displayName": {
+      "description": "Display Name that identifies this Query. It could be title or label.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of a query.",
+      "$ref": "../../type/basic.json#/definitions/markdown"
+    },
+    "version": {
+      "description": "Metadata version of the entity.",
+      "$ref": "../../type/entityHistory.json#/definitions/entityVersion"
+    },
+    "updatedAt": {
+      "description": "Last update time corresponding to the new version of the entity in Unix epoch time milliseconds.",
+      "$ref": "../../type/basic.json#/definitions/timestamp"
+    },
+    "updatedBy": {
+      "description": "User who made the query.",
+      "type": "string"
+    },
+    "href": {
+      "description": "Link to this Query resource.",
+      "$ref": "../../type/basic.json#/definitions/href"
+    },
+    "changeDescription": {
+      "description": "Change that lead to this version of the entity.",
+      "$ref": "../../type/entityHistory.json#/definitions/changeDescription"
+    },
+    "owner": {
+      "description": "Owner of this Query.",
+      "$ref": "../../type/entityReference.json",
+      "default": null
+    },
+    "duration": {
+      "description": "How long did the query took to run in seconds.",
+      "type": "number"
+    },
+    "users": {
+      "description": "List of users who ran this query.",
+      "type": "array",
+      "items": {
+        "$ref": "../../type/entityReference.json"
+      },
+      "default": null
+    },
+    "followers": {
+      "description": "Followers of this Query.",
+      "$ref": "../../type/entityReference.json#/definitions/entityReferenceList"
+    },
+    "vote": {
+      "description": "Users can vote up to rank the popular queries.",
+      "type": "integer",
+      "default": 0
+    },
+    "query": {
+      "description": "SQL Query definition.",
+      "$ref": "../../type/basic.json#/definitions/sqlQuery"
+    },
+    "checksum": {
+      "description": "Checksum to avoid registering duplicate queries.",
+      "type": "string"
+    },
+    "queryDate": {
+      "description": "Date on which the query ran.",
+      "$ref": "../../type/basic.json#/definitions/timestamp"
+    },
+    "tags": {
+      "description": "Tags for this SQL query.",
+      "type": "array",
+      "items": {
+        "$ref": "../../type/tagLabel.json"
+      },
+      "default": null
+    },
+    "queryUsedIn": {
+      "description": "Entities that are using this query",
+      "$ref": "../../type/entityReference.json#/definitions/entityReferenceList"
+    },
+    "deleted": {
+      "description": "When `true` indicates the entity has been soft deleted.",
```

Comment:
If soft delete is not supported, we can remove this deleted flag.

Reply:
removed